doc/version_history.rst

```diff
@@ -6,6 +6,21 @@
 Version History
 ###############
 
+=======
+v0.15.3
```

Comment:
Please fix the version history below at v0.15.1: there is no v0.15.1; I combined that entry into v0.15.0 before tagging v0.15.0 (it has all the fixes listed for the non-existent v0.15.1, making for a much neater history) and I missed that you re-introduced it when I reviewed your v0.15.2 PR. So: look at the version history for v0.15.0 and use that text up to v0.15.0, then use your text for v0.15.2 and now v0.15.3.

You could also note that v0.15.0 demands ts_xml 16 (a fact that I missed at the time), but I consider that a bug that you fixed in v0.15.2, so I'd be inclined to leave it alone and just note the fix in the v0.15.2 notes.

Reply:
I already tagged v0.15.2 so I'll stick with v0.15.3.